009_team_7_agent_design/009_team_7_agent_design.tex

```diff
@@ -1 +1,459 @@
+
+\subsubsection{Inter-team Comparison}
+\Cref{tab: Inter-team data} shows the total number of deaths for simulations of various combinations of agent teams. All simulations are run with a simulation period of 500 days, shuffle period of 7 days and 10 team 7 agents with 10 agents of one other team type. The table is further broken down into food available per agent (5, 10 and 15).
+
+\begin{table}
+    \begin{center}
+    \begin{tabular} { | m{4em} | m{6em} | m{4em} | m{4em} | m{4em} | }
+        \hline
+        \textbf{Agent Team} & \textbf{Food/Agent} & \textbf{Total Deaths} & \textbf{Team 7 Deaths} & \textbf{Other Deaths} \\
+        \hline
+        2 & 5 & 463 & 273 & 190 \\
+        \hline
+        2 & 10 & 149 & 80 & 49 \\
+        \hline
+        2 & 15 & 0 & 0 & 0 \\
+        \hline
+        3 & 5 & 347 & 183 & 164\\
+        \hline
+        3 & 10 & 73 & 53 & 20\\
+        \hline
+        3 & 15 & 0 & 0 & 0 \\
+        \hline
+        4 & 5 & 217 & 92 & 125\\
+        \hline
+        4 & 10 & 0 & 0 & 0 \\
+        \hline
+        4 & 15 & 0 & 0 & 0 \\
+        \hline
+        5 & 5 & 213 & 107 & 106 \\
+        \hline
+        5 & 10 & 5 & 5 & 0 \\
+        \hline
+        5 & 15 &  0 & 0 & 0 \\ 
+        \hline
+        6 & 5 & 459 & 136 & 323 \\
+        \hline
+        6 & 10 & 273 & 48 & 225 \\
+        \hline
+        6 & 15 & 126 & 24 & 102 \\
+        \hline
+    \end{tabular}
+    \end{center}
+    \caption{Comparison between agent teams and team 7.}
+    \label{tab: Inter-team data}
+\end{table}
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.4]{009_team_7_agent_design/Images/Chart.png}
+    \end{center}
+    \caption{Deaths vs Agent Teams.}
+    \label{fig: agent deaths}
+\end{figure}
+
+All team agents, bar team 6, were able to coexist with team 7 and achieve zero deaths for a food availability of 15 per agent, indicating that Team 6 agent was the most incompatible with our agent. Team 4 achieved zero deaths for food availability of 10 demonstrating a greater compatibility with team 7 and team 5's results are not too dissimilar.  
+
+\subsubsection{Conclusion}
+The agent designed by team 7 is unique in terms of the use of personalities to determine the general behaviour of the agent. These personality traits assigned allow it to mimic human behaviour as much as possible. The results presented above show that the team 7 can have a vastly different behaviours with each other agent, showing our compatibility with some agents and incompatibility with others. In short, the team 7 agent has been intricately designed to meaningfully engage in a huge variety of scenarios and attempt to accomplish its to major goals, to survive individually and collectively.
```

Comment:
```suggestion
The agent designed by team 7 is unique in terms of the use of personalities to determine the general behaviour of the agent. These personality traits assigned allow it to mimic human behaviour as much as possible. The results presented above show that team 7's agents can have vastly different behaviours with each other agent, showing our compatibility with some agents and incompatibility with others. In short, team 7 agents have been intricately designed to meaningfully engage in a huge variety of scenarios and attempt to accomplish major goals, to survive individually and collectively.
```

Reply:
Not sure if these are the right changes but this last paragraph seems a bit off.